.github/workflows/docker-release.yaml

```diff
@@ -384,9 +384,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cassandra-version: [5.0-alpha1]
+        cassandra-version: [5.0-alpha2]
```

Comment:
Did `alpha1` get pulled? I would expect to see both versions here.

```suggestion
        cassandra-version: [5.0-alpha1, 5.0-alpha2]
```

Reply:
No, it's still available. For 4.x versions, I think we just stopped publishing alphas/betas/rcs when the newest one came out. I can put alpha1 back in, but when 5.0 is GA, I'm thinking we probably stop publishing alphas/betas/rcs

Reply:
That's reasonable. Thanks!